javascript/javascript2/week3/question3.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <style>
+      #time-input {
+        padding: 10px;
+        font-size: 18px;
+        margin-right: 10px;
+      }
+
+      #start-button {
+        padding: 10px 20px;
+        font-size: 18px;
+        background-color: #4caf50;
+        color: white;
+        border: none;
+        cursor: pointer;
+      }
+
+      #feedback {
+        font-size: 20px;
+        font-weight: bold;
+        margin-top: 20px;
+      }
+    </style>
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Fastest Presser</h1>
+    <input type="text" id="time-input" />
```

Comment:
You could label this input, it is bit hard to figure out what to type in there. It could have a label like "How long you want to play (seconds)". This input should have a `type="number"` atribute on it and some min and max values :)